jobs/infra-ps5.yaml

```diff
@@ -55,3 +55,63 @@
 
               venv/bin/pip install ansible
               venv/bin/ansible-playbook jobs/infra/playbook-jenkins.yml --limit localhost --tags 'jenkins' -i jobs/infra/hosts
+
+- job:
+    name: 'infra-maintain-ps5-amd64-12'
+    description: |
+      Temp job for testing juju3. This is just like the other infra jobs,
+      but exposes the ability to change the juju snap on the ps5-12 node.
+    node: runner-ps5-amd64-12
+    project-type: freestyle
+    scm:
+      - k8s-jenkins-jenkaas
+    parameters:
+      - string:
+          name: JUJU_CHANNEL
+          default: '3.0/candidate'
```

Comment:
any reason to not use `3.1/stable` here?

Reply:
nah, no reason.  let's do it!